pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -386,3 +386,89 @@ def get_convergence(self):
     def get_fermi(self):
         pattern = r"Fermi energy:\s+(\d+\.\d+)\s+eV"
         return np.array(re.findall(pattern, self.log_main)).astype(float)
+
+
+class SphinxWavesParser:
+    """Class to read SPHInX waves.sxb files (HDF5 format)
+
+    Initialize with waves.sxb filename, or use load ()
+
+    """
+
+    def __init__(self, fname=None):
+        if fname is not None:
+            self.load(fname)
+
+    def load(self, filename):
+        """Load a waves.sxb file (HDF5 format)
+
+        filename: file name
+        """
+        self.wfile = h5py.File(filename)
+        self._eps = None
+        self._read()
+
+    # Internal: check that wfile is set
+    def _check_loaded(self):
+        if not isinstance(self.wfile, h5py.File):
+            raise "No waves file loaded"
+
+    def _read(self):
+        self._check_loaded()
+        # load various dimensions
+        self.mesh = self.wfile["meshDim"][:]
+        self.Nx, self.Ny, self.Nz = self.mesh
```

Comment:
```suggestion
        self.Nx, self.Ny, self.Nz = self.wfile["meshDim"][:]
```

Reply:
There is another [repo](https://github.com/eisenforschung/EXTRA_FIM/blob/main/EXTRA_FIM/main.py), based on sphinx wavefunctions, which uses some of these properties. Maybe in future when we make changes to that, I can change it.

Reply:
Ah you mean `self.mesh` is also used elsewhere? Then I think it should also be a property

Reply:
yep ok.